apps/api/v2/src/ee/bookings/controllers/bookings.controller.ts

```diff
@@ -243,6 +246,23 @@ export class BookingsController {
     throw new InternalServerErrorException("Could not create instant booking.");
   }
 
+  @Post("/no-show")
+  @Permissions([BOOKING_WRITE])
+  @UseGuards(ApiAuthGuard)
```

Comment:
Completely unaware of it, so I want to know why we are using ApiAuthGuard for some endpoints and not for others within the controller